assets/sass/_event-list.scss

```diff
@@ -0,0 +1,39 @@
+.pulumi-event-list-container {
+    @each $filter-name in "all", "featured", "upcoming", "videos", "pulumitv" {
+        ##{$filter-name}:target {
+            @if $filter-name != "all" {
+                ~ .event-list-filter li[data-filter-name="all"] a {
+                    @apply text-gray-700;
+                }
+
+                ~ h3[data-filter-title="all"] {
+                    @apply hidden;
+                }
+            }
+
+            ~ .event-list-filter li[data-filter-name="#{$filter-name}"] a {
+                @apply text-blue-500;
+            }
+
+            ~ h3[data-filter-title="#{$filter-name}"] {
+                @apply block;
+            }
+
+            // To support intially loading all the items on the page we first need
+            // to hide all the elements and then set the specific items to be visible.
+            ~ ul li {
```

Comment:
You're in the loop here, so this rule will simply be repeated in the generated CSS. Since I think this means "hide all elements" -- i.e., there isn't any reference to `filter-name` here -- you can pull it out of the loop and position it before the loop to make it the default (unless I'm misunderstanding).

Reply:
I'll try and make the comment clearer but this piece needs to be here because when the filter changes we need to first hide all the elements and then only show the ones we want. Because the filter triggers off hashes in the URL we need to handle the case where there is no hash in URL and in the case of no hash we show all the items, meaning that the elements are not rendered initially with a `hidden` class. Because there is no hidden class when the element is initially rendered when need to apply the hidden class when the filter changes.

TL;DR When the page initially renders all the elements are visible when the filter changes we need to hide the elements that are not part of the selected filter.